src/cmd/singularity/cli/pull.go

```diff
@@ -62,7 +61,7 @@ var PullCmd = &cobra.Command{
 		case ShubProtocol:
 			libexec.PullShubImage(image, uri, force)
 		default:
-			sylog.Errorf("Not a supported URI")
+			libexec.PullOciImage(image, uri, force)
```

Comment:
we are moving the `FAIL` to the `libexec.PullOciImage()` ?I think Default should still be fail, not a supported URI

Reply:
It's good enough for the time being. It will fail with an `unsupported transport` error, and should tell the user that the URI isn't supported